### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by gabriela99**
Most of what has been worked on in dashboard is already in development, but generally here I added in create.html (I'll continue working on it in a new branch), and I added the dashboard app. I've pulled the changes from development and feat_resource so it shouldn't conflict. Merging this so I can close it.

### CODE REVIEW

File: dashboard/templates/dashboard/create.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                    <option>Article</option>
+                    <option>Software/Tools</option>
+                    <option>Slides</option>
+                    <option>Audio/Podcast</option>
+                    <option>Library</option>
+                    <option>Database</option>
+                    <option>Blog</option>
+                    <option>Other</option>
+                </select>
+            </div>
```
Comment (dani-santos-code) [inline dashboard/templates/dashboard/create.html:69]: For this, I would loop over the info we have from the database!

**Review by dani-santos-code — APPROVED** [review]
Overall it looks good. I would just change a bit the select tag and use a loop to display the option tags with the values from the database.

File: dashboard/templates/dashboard/create.html (hunk 1 above)
Comment (gabriela99) [reply dashboard/templates/dashboard/create.html:69]: Great idea. I'll be editing it in a new branch so it's still rough, but I'll make sure to add that in.

**Comment by faymarie** [comment]
Apart from creating select options dynamically, it looks fine to me.

**Pull request merged**